src/spikeinterface/extractors/neoextractors/neobaseextractor.py

```diff
@@ -200,13 +200,13 @@ def __init__(
             use_names_as_ids = False
 
         stream_channels = self.neo_reader.header["signal_streams"]
-        stream_names = list(stream_channels["name"])
-        stream_ids = list(stream_channels["id"])
+        stream_names = stream_channels["name"].tolist()
+        stream_ids = stream_channels["id"].tolist()
 
         if stream_id is None and stream_name is None:
             if stream_channels.size > 1:
                 raise ValueError(
-                    f"This reader have several streams: \nNames: {stream_names}\nIDs: {stream_ids}. \n"
+                    f"This reader have several streams: \nstream_names: {stream_names}\stream_ids: {stream_ids}. \n"
```

Comment:
```suggestion
                    f"This reader have several streams: \n`stream_names`: {stream_names}\`stream_ids`: {stream_ids}. \n"
```
we were going to add ticks to let people know these are arguments in the package :)

Reply:
Yes!